lib/svg.dart

```diff
@@ -182,11 +164,16 @@ class SvgPicture extends StatelessWidget {
     this.semanticsLabel,
     this.excludeFromSemantics = false,
     this.theme = const SvgTheme(),
+
+    /// The [ColorFilter] to apply to the picture. An easy way to add color.
+    /// e.g. `ColorFilter.mode(Colors.red, BlendMode.srcIn)`
```

Comment:
Here and elsewhere: docs should contain complete sentences thatstart with a capital letter and end with a period.

It would be great to expand this a bit to mention that it is also possible to use the `ColorMapper` property which avoids filtering at runtime and instead replaces colors at parse time.